testing/crg_config.yaml

```diff
@@ -0,0 +1,27 @@
+servers:
+  - server_name: Centre for Genomic Regulation
+    base_url: https://genome.crg.cat/rnaget/
+    token: abcdefuvwxyz
+    implemented:
+        projects: true
+        studies: true
+        expressions: true
+        continuous: false
+    
+    projects:
+      - id: E-MTAB-5200
+        filters:
+          name: Pancancer Analysis of Whole Genomes
+      - id: E-GEOD-81547
+        filters:
+          name: Single cell transcriptome analysis of human pancreas
+    
+    studies:
+      - id: E-MTAB-5200-ST0
+        filters:
+          name: Pancancer Analysis of Whole Genomes
+    
+    expressions:
+      - id: 8beada7b93d5e55aa557138b39c6f930
+        filters:
+          tags: cancer 
```

Comment:
`format` is now a required filter for each expression, as format must be specified in the `/expressions/search` endpoint. The expression object definition should look like this:

```
expressions:
  - id: 8beada7b93d5e55aa557138b39c6f930
    filters:
    tags: cancer
    format: tsv
```